CHANGELOG.md

```diff
@@ -5,12 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
 =======
 
 ## UNRELEASED
 
 ### **Added**
 
+### **Changed**
+- modified `fsx-luster` module to auto-import data if configured
```

Comment:
little typo

Reply:
On it...and need to add more context to the README